pytest_splunk_addon_ui_smartx/base_test.py

```diff
@@ -83,6 +87,17 @@ def __init__(
                         firefox_options=self.get_local_firefox_opts(headless),
                         log_path="selenium.log",
                     )
+                elif selenium_host:
+                    options_firefox = webdriver.FirefoxOptions()
+                    options_firefox.add_argument("--ignore-ssl-errors=yes")
+                    options_firefox.add_argument("--ignore-certificate-errors")
+                    options_firefox.add_argument("--disable-dev-shm-usage")
+
+                    self.browser = webdriver.Remote(
+                        command_executor=f"{selenium_host}:4444/wd/hub",
+                        options=options_firefox,
```

Comment:
There were much more options used in the Saucelabs-based solution (including screen size and other options). How screen size is being controlled with Selenium-based solution?

Reply:
You are right, should we add `screenResolution` to our new configuration?

Reply:
Default resolution is [1360x1020](https://github.com/SeleniumHQ/docker-selenium/blob/trunk/README.md#setting-screen-resolution) and can be changes via env vars when instantiating selenium container.
I don't think we need to bother about that without any specific requirements.
If needed, can be adjusted on k8s-manifests.